new/property.md

```diff
@@ -0,0 +1,206 @@
+---
+RFC: unassigned
+Author: John Keiser <jkeiser@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Resource Properties
+
+We add `property` DSL to resources, similar to (and interoperable with) LWRP `attribute`.
```

Comment:
My reasoning was that `property` is a thing a noun (resource) has, and `parameter` is a thing a verb (function or action) has. I'm not crazy attached, but I *do* have that slight preference: it'll be a common enough word that it will flavor peoples' orientation towards what a resource is--whether it is a noun describing desired state or a verb describing an action that needs to be taken.  If you, @someara and @danielsdeleo would like the latter, I'll go there.